commands/config.ts

```diff
@@ -1,17 +1,29 @@
 import { Argv } from 'yargs';
-import { addConfigOptions, addGlobalOptions } from '../lib/commonOpts';
 import { i18n } from '../lib/lang';
-import * as set from './config/set';
-import * as migrate from './config/migrate';
+import set from './config/set';
+import migrate from './config/migrate';
+import { makeYargsBuilder } from '../lib/yargsUtils';
+import { YargsCommandModuleBucket } from '../types/Yargs';
 
-export const command = 'config';
-export const describe = i18n('commands.config.describe');
-
-export function builder(yargs: Argv): Argv {
-  addConfigOptions(yargs);
-  addGlobalOptions(yargs);
+const command = 'config';
+const describe = i18n('commands.config.describe');
 
+function configBuilder(yargs: Argv): Argv {
   yargs.command(set).command(migrate).demandCommand(1, '');
 
   return yargs;
 }
+
+const builder = makeYargsBuilder(configBuilder, command, describe);
```

Comment:
This will make reviews slightly more annoying (sorry), but I tried to clean up the areas that I was touching while adding this feature. We need to make these changes eventually anyways.